documentation/classes_and_modules.md

```diff
@@ -49,6 +51,12 @@ A proxy also has a repaating async timer task for handling watchdog and acknowle
 
 Proxy has to child classes: SiteProxy and SupervisorProxy.
 
+### SiteProxy
+A SiteProxy represents a connection from a Supervisor to a remote Site. It provides methods for sending commands and requesting status from the connected site.
+
+### TrafficLightControllerProxy
```

Comment:
To be consistent with other names, e.g. TrafficControllerSite, let's call the class TrafficControllerProxy